.github/lint-and-build.yml

```
@@ -0,0 +1,32 @@
+    branches: [main]
+  pull_request:
+    branches: [main]
+
+jobs:
+  lint-and-build:
+    runs-on: ubuntu-latest
+  strategy:
+    matrix:
+      project: [api-gateway, prompad-ui]
+  defaults:
+      run:
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 4 but found 6

(indentation)